libraries/opensk/src/ctap/mod.rs

```diff
@@ -548,6 +549,24 @@ impl<E: Env> StatefulPermission<E> {
             Err(Ctap2StatusCode::CTAP2_ERR_NOT_ALLOWED)
         }
     }
+
+    /// Returns the large blob state. If none is hold, create and hold a new state.
+    pub fn large_blob(&mut self, env: &mut E, channel: Channel) -> &mut LargeBlobState {
+        self.clear_timer(env);
+        if let Some(StatefulCommand::LargeBlob(_)) = &self.command_type {
+        } else {
+            self.set_command(
+                env,
+                StatefulCommand::LargeBlob(LargeBlobState::default()),
+                channel,
+            );
+        }
+        if let Some(StatefulCommand::LargeBlob(ref mut large_blob_state)) = &mut self.command_type {
```

Comment:
I don't think you need the `ref mut` here. My understanding is that it should be automatic.

Reply:
Nice, it is indeed unnecessary.